frontend/src/components/TabNavigation.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from "@emotion/styled";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { Nav } from "react-bootstrap";
+import { IntlShape, useIntl } from "react-intl";
+import BreadcrumbItem from "./BreadcrumbItem";
+
+const StyledNav = styled(Nav)`
+  margin: 1rem 0;
+
+  .nav-item {
+  }
+
+  .nav-link {
+    &.active {
+    }
+  }
+`;
+
+export interface NavigationTab<T = undefined> {
+  url: string;
+  title: (intl: IntlShape, args: T) => string;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-constraint
+const TabNavigation = <T extends unknown = undefined>({
```

Comment:
for my own understanding, why do we need to disable this / do what the linter says we shouldn't? 😄

Reply:
I will disable it for all tsx files because you need the extends unknown in tsx files as it otherwise conflicts with the xml syntax ..